### PR TITLE
Convert to ts

### PR DESCRIPTION
**Pull request by caiyan1**
Hi，our team need one version which support ts，can you publish one ? Thank you very much !

### CODE REVIEW

**Comment by xeolabs** [comment]
Thanks!

File: src/viewer/scene/math/rtcCoords.js [hunk 1]
```diff
@@ -32,7 +32,10 @@ const createRTCViewMat = (function () {
  *
  * Given a double-precision World-space position, returns a double-precision relative-to-center (RTC) center pos
  * and a single-precision offset fom that center.
+<<<<<<< HEAD
```
Comment (Amoki) [inline src/viewer/scene/math/rtcCoords.js:35]: Merge artifact

File: src/viewer/scene/math/rtcCoords.js [hunk 2]
```diff
@@ -66,7 +69,10 @@ function worldToRTCPos(worldPos, rtcCenter, rtcPos) {
  *
  * When computing the RTC position, this function uses a modulus operation to ensure that, whenever possible,
  * identical RTC positions are reused for different positions arrays.
+<<<<<<< HEAD
```
Comment (Amoki) [inline src/viewer/scene/math/rtcCoords.js:72]: Merge artifact

File: src/viewer/scene/math/rtcCoords.js [hunk 3]
```diff
@@ -117,7 +126,10 @@ function rtcToWorldPos(rtcCenter, rtcPos, worldPos) {
 /**
  * Given a 3D plane defined by distance from origin and direction, and an RTC center position,
  * return a plane position that is relative to the RTC center.
+<<<<<<< HEAD
```
Comment (Amoki) [inline src/viewer/scene/math/rtcCoords.js:129]: Merge artifact

**Comment by xeolabs** [comment]
I just realised that this PR includes a ````.gitignore```` that excludes the ````./dist```` directory. We can't do that because it will break the live demos, which rely on ````./dist````.

**Comment by xeolabs** [comment]
Bit confused here TBH. I've merged this PR into master but not seeing any of it - eg no tsconfig.json.

**Pull request merged**